draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -140,7 +140,7 @@ Specifically an ICE agent using an interface with both IPv4 and IPv6 addresses M
 ICE Candidate Processing {#processing}
 ----------------------------
 
-For any remote host ICE candidate received by the ICE agent, the following procedure is used:
+For any remote ICE host candidate received by the ICE agent, the following procedure is used:
```

Comment:
I think we should just drop 'host' entirely here. We won't worry about the type in this stage.